### PR TITLE
[DEV APPROVED] 9014 SDC links for LBTT and LTT

### PR DESCRIPTION
**Pull request by davidtrussler**
[TP9014](https://moneyadviceservice.tpondemand.com/restui/board.aspx?#page=userstory/9044&appConfig=eyJhY2lkIjoiOThBNEI1MURGODY0REREODQ4Qjk2MkE1RUY4NTRBMTIifQ==&searchPopup=userstory/9014)

This PR updates the Stamp Duty Calculator to reference the new tools for Scotland And Wales and make some associated copy changes.

Additionally there are some updates to the existing Wales/Scotland tools to ensure consistency across all three tools

Current:

image

Updated: 

image

### CODE REVIEW

File: config/locales/stamp_duty.cy.yml [hunk 1]
```diff
@@ -26,6 +24,21 @@ cy:
...
+        body: "Rhaid talu Treth Stamp ar eiddo yn Lloegr a Gogledd Iwerddon o hyd"
+        link: "/cy/tools/prynu-ty/cyfrifiannell-treth-stamp"
+      wales:
+        title: Cyfrifwch Dreth Trafodiadau Tir ar gyfer Cymru
+        body: Mae Treth Trafodiadau Tir ac (LTT) wedi disodli Treth Stamp yng Nghymru.
+        link: /cy/tools/prynu-ty/land-transaction-tax-calculator-wales
+      scotland:
+        title: Cyfrifwch Dreth Trafodiadau Tir ac Adeiladau ar gyfer yr Alban
+        body: Mae Treth Trafodiadau Tir ac Adeiladau (LBTT) wedi disodli Treth Stamp yn yr Alban.
+        link: /cy/tools/prynu-ty/land-and-buildings-transaction-tax-calculator-scotland
```
Comment (tomas-stefano) [inline config/locales/stamp_duty.cy.yml]: Can you change to use the links in welsh?
scotland:
   link: "/cy/tools/prynu-ty/cyfrifiannell-treth-stamp"

wales:
- "/cy/tools/prynu-ty/cyfrifiannell-treth-trafodiadau-tir-cymru"

Comment (davidtrussler) [reply config/locales/stamp_duty.cy.yml]: @tomas-stefano OK, I have changed those. (N.B. "cyfrifiannell-treth-stamp" is for en/ni, "cyfrifiannell-treth-trafodion-tir-ac-adeiladau-alban" for scotland)

File: app/views/mortgage_calculator/stamp_duties/_links_to_other_tools.html.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  <a class="elsewhere__item promo__link" href="<%= I18n.t('stamp_duty.elsewhere.wales.link') %>">
+    <div class="elsewhere__item-content">
+      <h3 class="promo__heading"><%= I18n.t('stamp_duty.elsewhere.wales.title') %></h3>
+      <p class="promo__content"><%= I18n.t('stamp_duty.elsewhere.wales.body') %></p>
+    </div>
+    <svg xmlns="http://www.w3.org/2000/svg" class="svg-icon svg-icon--arrow-right elsewhere__arrow" focusable="false">
+      <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-icon--arrow-right"></use>
+    </svg>
+  </a>
+</div>
```
Comment (margOnline) [inline app/views/mortgage_calculator/stamp_duties/_links_to_other_tools.html.erb]: There is a lot of duplicated code in this view and in the views of the same name in the land-and-transaction-tax and land-and-buildings-transaction-tax folders. Is it possible to use the same markup and pass variables to it? For example, this file could be in the shared folder for easier access and the relevant controllers could pass the countries to the calling view.

Comment (davidtrussler) [reply app/views/mortgage_calculator/stamp_duties/_links_to_other_tools.html.erb]: @margOnline I've done something along the lines you have suggested: could you have a look at the last commit on this?

Comment (margOnline) [reply app/views/mortgage_calculator/stamp_duties/_links_to_other_tools.html.erb]: This is good, but there are still 2 'links_to_other_tools' partials. Is there a time limit on this pr? I can have a look at using just one partial.

Comment (davidtrussler) [reply app/views/mortgage_calculator/stamp_duties/_links_to_other_tools.html.erb]: @margOnline My fault, missed this first time: I deleted land_and_buildings_transaction_taxes/_links_to_other_tools.html.erb and land_transaction_taxes/_links_to_other_tools.html.erb the original one, stamp_duties/_links_to_other_tools.html.erb, was no longer being used but i forgot to delete it. I have now deleted that as well. All are now replaced with shared/_links_to_other_tools.html.erb.

**Comment by margOnline** [comment]
nice work 👍

**Comment by johnplayer1982** [comment]
:+1:

**Pull request merged**